CHANGELOG.md

```diff
@@ -14,22 +14,24 @@ We use the following categories for changes:
 
 ## [Unreleased]
 
+## [0.10.0] - 2022-02-17
+
 ### Added
 - Add Prometheus metrics support for Tracing [#1102]
```

Comment:
You've missed https://github.com/timescale/promscale/pull/1152

Reply:
That is not required as `Add Prometheus metrics support for Tracing` covers it.

Reply:
Sure but then we should reference the PR at least :)